app/models/resort.rb

```diff
@@ -1,4 +1,16 @@
 class Resort < ApplicationRecord
   has_many :bookings
   belongs_to :user
+
+  validates :name, presence: true
+  validates :address, presence: true
+  validates :description, presence: true
+  validates :phone_number, presence: true, numericality: { only_integer: true }
```

Comment:
- [ ] Optional
    - Phone numbers can vary in format (e.g., including dashes, parentheses, or spaces), and using the only_integer option might not be sufficient to handle all valid phone number formats. So consider going forward the validation might change.

Reply:
Thanks, I will try to include some other format.